### PR TITLE
Initial Travis CI file

### PR DESCRIPTION
**Pull request by Krisselchen**
added initial Travis CI file with substructure for Python and Docker.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+sudo: required
+
+language: python
+
+python:
+  - "2.6"
+  - "2.7"
```
Comment (Drakulix) [inline .travis.yml]: I think we have explicitly stated not to support python 2, so you can remove these.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  - "3.2"
+  - "3.3"
+  - "3.4"
+  - "3.5"
+  - "3.6"
```
Comment (Drakulix) [inline .travis.yml:10]: We probably also don't need to support every python3 version, but thats open to discussion.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+install: "pip install flask"
+
+services:
+  - docker
```
Comment (Drakulix) [inline .travis.yml:15]: Please add any `script`-key to make travis pass. (`true` should be enough as a command)

**Review by Drakulix — CHANGES_REQUESTED** [review]
Thanks! I have requested some smaller changes. Otherwise this is good for merge.

File: .travis.yml (hunk 2 above)
Comment (Krisselchen) [reply .travis.yml:10]: I was not sure about it. Therefore I also took those. It is reduced to 3.x now.

File: .travis.yml (hunk 3 above)
Comment (Krisselchen) [reply .travis.yml:15]: Done. Travis is passing now.

**Review by Drakulix — APPROVED** [review]
Looks good!

**Pull request merged**